csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.shared.cs

```diff
@@ -161,50 +161,6 @@ public static SessionOptions MakeSessionOptionWithTensorrtProvider(OrtTensorRTPr
                 throw;
             }
         }
```

Comment:
is it ok to remove methods from the public API? it might break some code. the C API ROCm EP functions should already return an error now if called.